src/main/java/com/nylas/HostedAuthentication.java

```diff
@@ -46,7 +47,14 @@ public AccessToken fetchToken(String authorizationCode) throws IOException, Requ
 		params.put("grant_type", "authorization_code");
 		params.put("code", authorizationCode);
 
-		return application.getClient().executePost(null, tokenUrl, params, AccessToken.class);
+		// Send JSON accept header to force JSON response from this endpoint
+		Request request = new Request.Builder()
+				.url(tokenUrl.build())
+				.addHeader("Accept", "application/json")
```

Comment:
This is a really common header. Can we make the key and value java constants?

Reply:
@quzhi1 Good call, added constants for Header fields and values!